routes/controllers/channel.controller.js

```diff
@@ -148,9 +140,10 @@ exports.updateChannel = async (req, res, next) => {
 
     switch (state) {
       case 'voting': {
-        const user = players.find((player) => {
-          player.userId.toString() === userId;
-        });
+        const user = targetChannel.players.find(
+          (player) => player._id.toString() === playerId,
```

Comment:
```suggestion
          ({ _id }) => _id.toString() === playerId,
```
이렇게 하면 좀 더 보기 좋을 것 같아요!